runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -124,4 +124,24 @@ default boolean useSmallFont()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "infoBoxVertical",
+		name = "Display infoboxes vertically",
+		description = "Toggles the infoboxes displaying vertically"
+	)
+	default boolean infoBoxVertical()
```

Comment:
I don't like this very much if this is just workaround to have before resizeable overlays, as I will remove this and the infobox layout will be always horizontal just like other clients have it, but I guess for now it is fine, hopefully people will not get used to it.